src/protocol.rs

```diff
@@ -263,16 +291,21 @@ pub struct CommandLineArgs {
     /// Path to private key.
     #[arg(long, env = "COLINK_CLIENT_KEY")]
     pub key: Option<String>,
+
+    /// Disable automatically stop.
+    #[arg(long, env = "COLINK_DISABLE_AUTO_STOP")]
```

Comment:
similar here: why use a boolean to specify the negative option? maybe `COLINK_STOP_WHEN_SERVER_DISCONNECTED`?